alephium/contracts/token_bridge/attest_token_handler.ral

```diff
@@ -2,15 +2,15 @@ Contract AttestTokenHandler(
   governance: Governance,
   localTokenBridge: TokenBridge,
   targetChainId: U256,
-  targetTokenBridge: TokenBridge,
+  targetTokenBridgeId: ByteVec,
   mut receivedSequence: U256,
   isLocalHandler: Bool
 ) extends Constants(), TokenBridgeConstants() {
   @using(updateFields = true)
   fn parseAttestToken(vaa: ByteVec) -> (ByteVec, ByteVec, ByteVec, U256, U256) {
     let (emitterChainId, vaaTargetChainId, emitterAddress, msgSequence, payload) = governance.parseAndVerifyVAA(vaa, false)
     assert!(emitterChainId == targetChainId, ErrorCodes.InvalidEmitChainId)
-    assert!(emitterAddress == tokenId!(targetTokenBridge), ErrorCodes.InvalidEmitAddress)
+    assert!(emitterAddress == targetTokenBridgeId, ErrorCodes.InvalidEmitAddress)
```

Comment:
Q: the emitter address is contract id, not address?

Reply:
This is a naming convention in wormhole, where all emitters on the chain are named `emitterAddress`. So I have kept this naming. And in Alephium, the emitter address is the contract id.